ui/selectors/selectors.js

```diff
@@ -1778,7 +1779,9 @@ export function getShowRecoveryPhraseReminder(state) {
  * @returns Number of unapproved transactions
  */
 export function getNumberOfAllUnapprovedTransactionsAndMessages(state) {
```

Comment:
can we also rename [this variable](https://github.com/MetaMask/metamask-extension/blob/03b5d5ef766f05f20ddf5efa90b129f7944d5af4/ui/pages/routes/routes.container.js#L137) to `numberOfAllUnapprovedTransactionsAndMessages`?